sdk/core/src/main/java/com/klaviyo/core/config/KlaviyoConfig.kt

```diff
@@ -11,7 +11,7 @@ import com.klaviyo.core.Registry
 import com.klaviyo.core.networking.NetworkMonitor
 
 /**
- * Exception that is thrown when the the Klaviyo API token is missing from the config
+ * Exception that is thrown when the Klaviyo API token is missing from the config
```

Comment:
can't help fixing grammar things when i see them

Reply:
is there a linter that helps with this?

Reply:
Good Q, grammar problems are highlighted in a shade of red in my IDE color theme, so it always catches my eye.

Reply:
we should find a way to share some IDE configs